ui/src/app/store/device/selectors.ts

```diff
@@ -4,6 +4,7 @@ import type { RootState } from "../root/types";
 
 import { DeviceMeta } from "app/store/device/types";
 import type { Device, DeviceState, DeviceStatus } from "app/store/device/types";
+import { FilterDevices } from "app/store/device/utils";
 import { generateBaseSelectors } from "app/store/utils";
 
 const searchFunction = (device: Device, term: string) =>
```

Comment:
I think we can remove this function now as it'll be overwritten by the search selector below.